### DISCUSSION TITLE
OME Missing stream start/end webhooks

### DISCUSSION THREAD

**Discussion by hernanrz** [OvenMediaLabs/OvenMediaEngine#1420] [Help]
We've noticed that when OBS experiences rapid disconnect/reconnect attempts, OME will miss some of the stream start/end webhooks, this specially affects the webhooks for when a stream ends

Server.xml: https://gist.github.com/hernanrz/40243c766094e5ffd374397dcfc2a57e

**Comment by getroot** [comment]
Are there any logs related to webhook failures in the log file?

**Reply by naanlizard** [reply]
The origin logs here may be helpful - https://github.com/AirenSoft/OvenMediaEngine/discussions/1421

We pushed a fix for the failing webhook callback, which we think shouldn't have caused this, but haven't had time to test whether we're missing callbacks since the fix. I'll test that soon
